web/modules/custom/pulses_exchange_rate/pulses_exchange_rate.routing.yml

```diff
@@ -0,0 +1,6 @@
+pulses_exchange_rate.settings:
+  path: '/pulses_exchange_rate/settings'
```

Comment:
Where the user can find this link on the UI?

Reply:
Link to module settings was removed from structure menu. Now you can find link for configure form in module description (extend menu list)